clients/web/src/lib/api/queries/BannedPlayers.store.ts

```diff
@@ -0,0 +1,120 @@
+import type {OperationResult} from "@urql/core";
+import {gql} from "@urql/core";
+import {LiveQueryStore} from "$lib/api/core/LiveQueryStore";
+
+enum MemberRestrictionType {
+    QUEUE_BAN = "QUEUE_BAN",
+    RATIFICATION_BAN = "RATIFICATION_BAN",
+}
+
+interface MemberProfile {
+    name: string;
+}
+
+interface Member {
+    profile: MemberProfile;
+}
+
+export interface MemberRestrictionEvent {
```

Comment:
Add a `restriction` property and type it as MemberRestriction so it's not duplicating the restriction properties and is handled by the resolver without extra work.